tests/kani/ValidValues/write_invalid.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Z valid-value-checks
+
+//! Check that Kani can identify UB after writing an invalid value.
+//! Writing invalid bytes is not UB as long as the incorrect value is not read.
+//! However, we over-approximate for sake of simplicity and performance.
+
+use std::num::NonZeroU8;
+
+#[kani::proof]
+#[kani::should_panic]
+pub fn write_invalid_bytes_no_ub() {
```

Comment:
This test seems to be subsumed by the next one.

Reply:
I actually would prefer keeping it. In an ideal world, this test should not panic. This test shows that the UB detection for this case today is an over-approximation. I could turn it into a fixme test, but it's not really a bug at this point, it is a decision made by design.